CONTRIBUTING.md

```diff
@@ -257,7 +257,7 @@ will allow us to reach you directly with project announcements.
   - New features should be added only if they are necessary or generally useful.
   - Introduction of language constructions not currently used in MFEM should be
     justified and generally avoided (so we can build on cutting-edge systems).
-  - We prefer basic C++ and the C++03 standard, to keep the code readable by
+  - We prefer basic C++ and the C++11 standard, to keep the code readable by
```

Comment:
I vote for reverting this note. Even though C++11 is now enabled so we can do the lambda tricks, I think we should still stick to simple C++ in most of the code. The fact that the code looks like it's from 1995 is a good thing, because everybody can understand it, and IMO it's one of the selling points of MFEM.

Reply:
Ok, reverted